### ISSUE TITLE
migration instructions when coming from ES exporter

### ISSUE THREAD

**Issue by rursprung** [aiven/prometheus-exporter-plugin-for-opensearch#47]
is there a list of things (relevant for consumers) which changed between the original ES plugin (on 7.10.2) and this one? i can see that starting with #10 the default prefix is now `opensearch_` and not `es_` anymore (but can be configured). it might be good if a small migration guide (ES 7.10.2 -> OpenSearch 1.x -> OpenSearch 2.x) could be added to the README.
thanks!

**Comment by lukas-vlcek** [comment]
Good idea. Right now the migration should be simple (except the prefix as you pointed out).

**Comment by ervikrant06** [comment]
For opensearch 2.1.0 version, apart from prefix I see many metrics not showing up in-comparison to ES exporter.

Ex: all health related metrics. 
~~~
elasticsearch_cluster_health_active_primary_shards
~~~

**Comment by lukas-vlcek** [comment]
Hi @ervikrant06 

Is it the same metric that is pulled from ClusterHealth response? See:
https://github.com/aiven/prometheus-exporter-plugin-for-opensearch/blob/8f24ffbf3fc0965027b360ff3cd6eb71d53c9a45/src/main/java/org/compuscene/metrics/prometheus/PrometheusMetricsCollector.java#L114

Regards,
Lukáš
